### ISSUE TITLE
Proposal to update imports on folder move incorrectly marked as duplicate

### ISSUE THREAD

**Issue by devuxer** [Microsoft/vscode#69435]
_Creating new issue due to existing issues being locked._

#51756 was marked as a duplicate for #50244, but this does not appear to be correct.

#50244 seems to *only* address the simple case of renaming a folder, not moving a folder to a different part of the directory tree.

I was badly burned by thinking this feature would work and had to revert and manually move hundreds of files. (My use case was moving several folders into a "features" folder.)

Please consider re-opening #51756.

**Comment by devuxer** [comment]
P.S. I'm noticing that even moving a group files from one folder to another can be buggy due to the "eventually consistent" way in which the import renames seem to occur. I had cases where I moved several files, hit Save All, noticed further changes coming in, hit Save All again, and finally ended up with one or two files in both the original folder and the destination folder. Subsequently, I noticed some or the imports had not properly updated.
